infrastructure/running-a-node/setup-full-node.md

```diff
@@ -1,4 +1,258 @@
+--blocks-pruning archive \
+--rpc-cors all \
+--rpc-methods safe
+```
+
+## Secure the RPC Server
+
+The node startup settings allow you to choose what to expose, how many connections to expose, and which systems should be granted access through the RPC server. Some settings you can configure to increase node security include:
+
+- **Limit usable methods** - with  the `--rpc-methods` flag. An easy way to set this to a safe mode is: 
+```bash
+--rpc-methods safe
+```
+- **Set maximum number of connections** - with the `--rpc-max-connections` flag. For example, to limit maximum connections to 200 use:
+```bash
+--rpc-max-connections 200
+```
+- **Set server access permissions** - with the `--rpc-cors` flag. By default, localhost and Polkadot.js can access the RPC server. To allow access from everywhere, you can use: 
+```bash
+--rpc-cors all
+```
+
+For a list of important flags when running RPC nodes, refer to the Parity DevOps documentation: [Important Flags for Running an RPC Node](https://paritytech.github.io/devops-guide/guides/rpc_index.html?#important-flags-for-running-an-rpc-node){target=\_blank}
+
+## Secure the WebSocket Port
+
+To securely access your WebSocket (WS) connection over an SSL-enabled connection (necessary for SSL-enabled developer consoles), you'll need to convert the WS connection to a secure WSS connection. You can complete this conversion using a proxy and an SSL certificate. For detailed steps on setting this up, refer to the [Set Up Secure WebSockets](TODO: add path){target=\_blank} guide.
+
+## Connect to the Node
+
+Open [Polkadot.js](https://polkadot.js.org/apps){target=\_blank} and click the logo in the top left to switch the node. Activate the **Development** toggle and input your node's domain or IP address. Remember to prefix with `wss://`, and if you're using the 443 port, append `:443` as follows:
+
+```bash
+wss://example.com:443
+```
+
+??? info "Adjustments for Kusama" 
+    The bash commands that are provided to run against your node use `polkadot` as the default chain. 
+
+    Use the `--chain` flag to set up a Kusama node. For example:
+
+    ```bash
+    polkadot --name "Your Node's Name" --chain kusama
+    ```
+
+    Depending on where your binary is located (in some cases, it may be located within the `target/release` folder), you would use the same binary for Polkadot as you would for Kusama or any other relay chain:
+
+    ```bash
+    polkadot --name "Your Node's Name"
+    ```
+
+Use the `--help` flag to determine which flags you can use when running the node. For example, if you want to connect to your node remotely, you'll probably want to use `--rpc-external` and `--rpc-cors all`.
+
+The syncing process will take a while, depending on your capacity, processing power, disk speed, and RAM. For example, the process can be completed on a $10 DigitalOcean droplet in ~36 hours. Once it is synced, you may find it in [Telemetry](https://telemetry.polkadot.io/#list/Polkadot){target=_blank}.
```

Comment:
This should be after the start up command as well, this isn't really related to Connect to the Node